### PR TITLE
Use argparse for oid reading, print accounts

### PR DESCRIPTION
**Pull request by chriskuehl**
slight improvement to your branch? @matthew-mcallister 

adds a more useful help message by using argparse, also prints the list of accounts when there are already accounts (and does a "press any key to continue..." so you actually see the message)

### CODE REVIEW

**Comment by matthew-mcallister** [comment]
Sure, seems like an improvement.

File: create/approve.py [hunk 1]
```diff
@@ -21,10 +22,10 @@
 
 TEMPLATE = dedent(
     """\
-    user_name: {empty}
+    user_name:
     group_name: {group_name}
     callink_oid: {callink_oid}
-    signatory: {empty}
+    signatory:
```
Comment (matthew-mcallister) [inline create/approve.py]: If you do this, `group_name`, `callink_oid`, and `email` will have trailing spaces after the colon when the editor opens up even if no OID is passed in, but `signatory` and `user_name` will not. I know there have been complaints that the YAML parser will fail if you accidentally leave out the space after the colon, so it seemed like a general improvement in addition to simple consistency.

Comment (matthew-mcallister) [inline create/approve.py]: The linter removes trailing spaces in the source, so I put some useless text after the colon in the string and removed it at run time.

**Comment by chriskuehl** [comment]
@matthew-mcallister makes sense, added it back

**Pull request merged**